support/utils/functions_test.go

```diff
@@ -36,3 +36,90 @@ func TestDedupe(t *testing.T) {
 		})
 	}
 }
+
+func TestToMapStringInterface_SuccessMap(t *testing.T) {
+	success := map[string]interface{}{
+		"test": true,
+	}
+	m, e := ToMapStringInterface(success)
+	assert.Equal(t, success, m)
```

Comment:
we want to check for the error first and if that check fails then return
then we want to check if `m` is the same as `success`

this applies to all tests. generally if the error assertion fails then the other value(s) are invalid anyway so we don't want false negatives or false positives.